test/unit/specs/components/Personalization/createOnClickHandler.spec.js

```diff
@@ -14,8 +14,11 @@ import createOnClickHandler from "../../../../../src/components/Personalization/
 describe("Personalization::createOnClickHandler", () => {
   let mergeMeta;
   let collectClicks;
-  const event = { type: "blah" };
-  const clickStorage = {};
+  const event = {
+    type: "blah",
+    mergeXdm: jasmine.createSpy()
+  };
+  const clickStorage = [];
```

Comment:
It doesn't seem like anything is testing the `merger` function that the `createOnClickHandler` creates and passes to `collectClicks`. Rather than just checking that the first argument to `collectClicks` is a function of any kind, you should instead call the function and ensure that `event.mergeXdm` and `mergeMeta` get called as a result.

Reply:
I'll add more coverage. Thank you